test/e2e/auth/audit.go

```diff
@@ -23,83 +23,624 @@ import (
 	"strings"
 
 	apiv1 "k8s.io/api/core/v1"
+	extensions "k8s.io/api/extensions/v1beta1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apiextensions-apiserver/test/integration/testserver"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apiserver/pkg/apis/audit/v1beta1"
 	"k8s.io/kubernetes/test/e2e/framework"
+	imageutils "k8s.io/kubernetes/test/utils/image"
 
+	"github.com/evanphx/json-patch"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+var (
+	watchTestTimeout int64 = 1
+	auditTestUser          = "kubecfg"
+
+	crd          = testserver.NewRandomNameCustomResourceDefinition(apiextensionsv1beta1.ClusterScoped)
+	crdName      = strings.SplitN(crd.Name, ".", 2)[0]
+	crdNamespace = strings.SplitN(crd.Name, ".", 2)[1]
+
+	watchOptions = metav1.ListOptions{TimeoutSeconds: &watchTestTimeout}
+	patch, _     = json.Marshal(jsonpatch.Patch{})
+)
+
 var _ = SIGDescribe("Advanced Audit [Feature:Audit]", func() {
```

Comment:
> `[Feature:Audit]`

Now that advanced auditing is enabled by default, and auditing is configured in all test environments, should we move this into the default test suite? @loburm @crassirostris

Reply:
That actually makes sense

@loburm Could you please make a PR?

Reply:
Sure. Do I need to update only this file: https://github.com/kubernetes/test-infra/blob/master/jobs/config.json

Or something else is required for moving the test to default test suite?

Reply:
You only need to remove `[Feature:Audit]` substring from the test name :)